activesupport/lib/active_support/configurable.rb

```diff
@@ -81,7 +81,7 @@ def configure
       #   User.new.allowed_access = true # => NoMethodError
       #   User.new.allowed_access        # => NoMethodError
       #
-      # Or pass <tt>instance_accessor: false</tt>, to opt out both instance methods.
+      # Or pass <tt>instance_accessor: false</tt>, to omit both instance methods.
```

Comment:
For consistencies sake, lets make sure all occurrences of these are fixed in the files you have changed.
For example this file has 2 more occurrences.